pants

```diff
@@ -12,8 +12,14 @@
 
 set -eou pipefail
 
-# NOTE: To use an unreleased version of Pants from the pantsbuild/pants master branch,
-#  locate the master branch SHA, set PANTS_SHA=<SHA> in the environment, and run this script as usual.
+# Source any custom bootstrap settings for Pants from PANTS_BOOTSTRAP if it exists.
+: ${PANTS_BOOTSTRAP:=".pants.bootstrap"}
+if [[ -f "${PANTS_BOOTSTRAP}" ]]; then
+  source "${PANTS_BOOTSTRAP}"
+fi
```

Comment:
**[Question, non-blocking]** I assume these are changes recommended by new pants versions, but that we have to tweak on our side because `pants` is saved to version control?

Reply:
yup - they update the install script in... semi-meaningful ways from time to time (?)